src/gmg-emulator/gmg-emulator.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
`netcore3.1` is fine but could also just go `net6.0` since its now LTS.

Reply:
I just picked the earliest thing I could make work on a modern Linux.